Changes.md

```diff
@@ -16,7 +16,7 @@ Features
   - Inference : Loads ONNX models and performs inference using an array of input tensors.
   - ImageToTensor : Converts images to tensors for use with the Inference node.
   - TensorToImage : Converts tensors back to images following inference.
-- VisualiserTool : Added tool to 3D viewer for visualising primitive variables on meshes.
+- VisualiserTool : Added tool to 3D viewer for visualising primitive variables on meshes. A list of available variable names for the current selection is available from the `dataName` context menu.
```

Comment:
This will need moving to the `1.5.x.x` section since we've released 1.5.2.0 now.

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/6181/commits/c78be9e302b870aec9724a4263d0765be2bd12cd